.github/dependabot.yml

```diff
@@ -0,0 +1,34 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates
+
+version: 2
+updates:
+  # repetitive because unfortunately Dependabot ticket for wildcard support still open:
+  # https://github.com/dependabot/dependabot-core/issues/2178
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+
+  - package-ecosystem: 'cargo'
+    directory: '/'
```

Comment:
If Dependabot runs for example cargo upgrade in the given directory, then listing the subdirectories would not be necessary. But I am not sure what Dependabot does exactly, so we may find we need to add the subdirectories.